src/IQFeed.CSharpApiClient/Streaming/Level1/Messages/UpdateSummaryMessage.cs

```diff
@@ -62,26 +62,26 @@ public class UpdateSummaryMessage : IUpdateSummaryMessage
         {
             var values = message.SplitFeedMessage();
-            var symbol = values[1];                                                                                                                            // field 2
-            double.TryParse(values[2], NumberStyles.Any, CultureInfo.InvariantCulture, out var mostRecentTrade);                                        // field 71
-            int.TryParse(values[3], NumberStyles.Any, CultureInfo.InvariantCulture, out var mostRecentTradeSize);                                           // field 72
-            DateTime.TryParseExact(values[4], UpdateMessageTimeFormat, CultureInfo.InvariantCulture, DateTimeStyles.None, out var mostRecentTradeTime);
-            int.TryParse(values[5], NumberStyles.Any, CultureInfo.InvariantCulture, out var mostRecentTradeMarketCenter);                                   // field 75
-            int.TryParse(values[6], NumberStyles.Any, CultureInfo.InvariantCulture, out var totalVolume);                                                   // field 7
-            double.TryParse(values[7], NumberStyles.Any, CultureInfo.InvariantCulture, out var bid);                                                    // field 11
-            int.TryParse(values[8], NumberStyles.Any, CultureInfo.InvariantCulture, out var bidSize);                                                       // field 13
-            double.TryParse(values[9], NumberStyles.Any, CultureInfo.InvariantCulture, out var ask);                                                    // field 12
-            int.TryParse(values[10], NumberStyles.Any, CultureInfo.InvariantCulture, out var askSize);                                                      // field 14
-            double.TryParse(values[11], NumberStyles.Any, CultureInfo.InvariantCulture, out var open);                                                  // field 20
-            double.TryParse(values[12], NumberStyles.Any, CultureInfo.InvariantCulture, out var high);                                                  // field 9
-            double.TryParse(values[13], NumberStyles.Any, CultureInfo.InvariantCulture, out var low);                                                   // field 10
-            double.TryParse(values[14], NumberStyles.Any, CultureInfo.InvariantCulture, out var close);                                                 // field 21
-            var messageContents = values[15];                                                                                                                  // field 80
-            var mostRecentTradeConditions = values[16];                                                                                                        // field 74
+            var symbol = values[1];                                                                         // field 2
+            var mostRecentTrade = FieldParser.ParseDouble(values[2]);                                       // field 71
```

Comment:
I'd prefer to keep the previous implementation if you dont mind because of the return value doesn't exist. Potentially it gets inlined but not sure.

Reply:
> @mrut2pac I think we are very close to merge this. Can we get on the same page for the next small refactor about passing dynamic fields in the constructor ?
> 
> I was thinking to have a Level1DynamicClient that supports your new Handler and instead of creating all these new properties in existing UpdateSummaryMessage we could have a new UpdateSummaryDynamicMessage and this one will be available in ILevel1Event as a generic for your new Handler. What do you think? That way, there's a clear separation. 
> 
> Any thoughts ?

Have you looked at my latest version? There is `IUpdateSummaryDynamicMessage` interface and generated object now implement it.

> Personally, I dont use dynamic fields so, I'd prefer to have the leanest possible path.
Are you using every single field from IUpdateSummaryMessage? If you don't you can still benefit by omitting the unneeded ones...

I will find time and create a new client to work with the new handler.